public/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+
+  <link rel="stylesheet" href="/app.css">
+    <meta charset="utf-8">
+    <title>09-skate-page</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav class="top-nav">
+        <button class="button"><span>SUBSCRIBE</span><i class="fa fa-circle"></i></button>
+       <nav class="top-nav-icons">
+        <i class="fa fa-ellipsis-v"></i>
+        <i class="fa fa-facebook-official"></i>
+        <i class="fa fa-rss-square"></i>
+        <i class="fa fa-twitter-square"></i>
+        <i class="fa fa-youtube-square"></i>
+        </nav>
+        </nav>
+        <nav class="red-button">
+          <button><i class="fa fa-minus"></i><span>SKBT NEWS</span><i class="fa fa-minus"></i></button>
```

Comment:
These nav items should be `a` not buttons. Because with the navigation bar we want to link the user to other pages.